### DISCUSSION TITLE
Discord server dedicated to MiniDexed ?

### DISCUSSION THREAD

**Discussion by Onosendaifr** [probonopd/MiniDexed#676] [Ideas]
A forum is quite oldschool to build a community, what about a Discord server dedicated to Mini Dexed  ?

There is only one for Micro Dexed (i've been there believing it was the one for Rasp Mini Dexed)

**Comment by probonopd** [comment]
Thanks for this suggestion, but Discord is not planned. 
GitHub is the official communication channel for this project.

Reasons:
* Discord needs yet another login
* Our community is in timezones all over the world, so we do asynchronous communication (an answer may take several days, not seconds)
* We want all communication preserved and searchable so that we don't have to answer the same questions time and again

I hope you can understand.

**Reply by Onosendaifr** [reply]
Sure ! Thanks for your answer.
